server/liveblog/items/items.py

```diff
@@ -131,3 +140,19 @@ def get(self, req, lookup):
             lookup['blog'] = ObjectId(lookup['blog_id'])
             del lookup['blog_id']
         return super().get(req, lookup)
+
+
+@drag_and_drop_blueprint.route('/api/archive/draganddrop/', methods=['POST'])
+def drag_and_drop():
+    data = request.get_json()
+    url = data['image_url']
+    mimetype = data['mimetype']
+
+    item_data = dict()
+    item_data['type'] = 'picture'
+    item_data['media'] = FileStorage(stream=fetch_url(url), content_type=mimetype)
+    archive_service = get_resource_service('archive')
+    archive_id = archive_service.post([item_data])[0]
+    archive = archive_service.find_one(req=None, _id=archive_id)
+
```

Comment:
It's not different to the superdesk archive. Just a different way of creating such an item. You send the url of the image to the server, and it is downloaded there. We can't add this to the archive code, as it's in superdesk. So, I created another endpoint to create an archive, reusing as much of the archive code as seems possible to me. The renditions are created by the archive code. If you can think of a better, cleaner way, let me know.